src/models/BaseLoginModel.js

```diff
@@ -41,6 +41,8 @@ function (Okta, Q, Enums) {
           }
           self.trigger('setTransactionError', err);
           self.trigger('error', self, err.xhr);
+          // Trigger an event once the previous 'error' event is handled
+          self.trigger('afterError', err);
```

Comment:
Once `self.trigger('error')` is called, the errorBox is rendered to the DOM. Since the `afterError` event is intended to give developers the ability to override the errorbox message, we need to trigger a special event here.